doozer/doozerlib/backend/konflux_image_builder.py

```diff
@@ -190,8 +190,12 @@ async def _start_build(self, metadata: ImageMetadata, build_repo: BuildRepo, dyn
             raise ValueError(f"[{metadata.distgit_key}] Unsupported arches: {', '.join(unsupported_arches)}")
         build_platforms = [self.SUPPORTED_ARCHES[arch] for arch in arches]
         pipelineruns_api = await self._get_pipelinerun_api(dyn_client)
+
+        # Replace 'openshift-4-18-ose-installer-terraform' with '4-18-ose-installer-terraform'
+        formatted_pipelinerun_name = component_name.replace("openshift-", "", 1) if component_name.startswith("openshift-") else component_name
+
         pipelinerun_manifest = self._new_pipelinerun(
-            f"doozer-build-{component_name}-",
+            formatted_pipelinerun_name,
```

Comment:
generate name should end with a trailing `-`

Reply:
Thanks for the catch! Updated.